docs/pages/versions/unversioned/sdk/media-library.md

```diff
@@ -42,9 +42,9 @@ Checks user's permissions for accessing media library. Alias for `Permissions.ge
 
 A promise that resolves to an object of type [CameraRollPermissionResponse](#medialibrarycamerarollpermissionresponse).
 
-### `MediaLibrary.presentLimitedLibraryPickerAsync()`
+### `MediaLibrary.presentPermissionsPickerAsync()`
 
-**Available only on iOS >= 14.** Allows the user to update the assets which your app will have access to. This method will display a system prompt only if permissions were `granted` and the `accessPrivileges` is equal to `limited`.
+**Available only on iOS >= 14.** Allows the user to update the assets which your app has access to. It displays a system modal only if permissions were `granted` and the `accessPrivileges` is equal to `limited`.
```

Comment:
```suggestion
**Available only on iOS >= 14.** Allows the user to update the assets which your app has access to. The system modal is displayed only if a user originally selected the `limited` access to the media library.
```
I would go with sth like this 🤔 
I guess there's still a better and simpler way to describe the actual behaviour than what I'm suggesting 😅